### PR TITLE
Team A Wiki Entry

### PR DESCRIPTION
**Pull request by anjalipemmaraju**

### CODE REVIEW

File: wiki/math/registration-techniques.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Registration Techniques in Robotics
```
Comment (smbryan) [inline wiki/math/registration-techniques.md:1]: I'd love to get this into the wiki. However, it doesn't follow any of our style guidelines and Jekyll is finicky as it is. Please [review the template](https://raw.githubusercontent.com/RoboticsKnowledgebase/roboticsknowledgebase.github.io/d595f7aac3f0000ce40d845113940171a4f8ff1a/_templates/template.md) and update this accordingly

**Comment by anjalipemmaraju** [comment]
Hi, I'm not sure which of the style guidelines this isn't following. If you could let me know that would be great.

File: wiki/math/registration-techniques.md [hunk 2]
```diff
@@ -1,110 +1,109 @@
 # Registration Techniques in Robotics
```
Comment (smbryan) [inline wiki/math/registration-techniques.md:1]: Missing the title block.

File: wiki/math/registration-techniques.md [hunk 3]
```diff
@@ -1,110 +1,109 @@
...
+Iterative Closest Point (ICP) is an algorithm to register two sets of point clouds iteratively. In each iteration, the algorithm selects the closest points as correspondences and calculates the transformation, i.e., rotation and translation (\\[ R \\] and \\[ t \\]. Although there are a number of techniques to calculate the transformation at each iteration, Horn's method remains a popular choice to do so. The figure below illustrates the registration of two different sets of point clouds(shown in pink and blue for distinction).
+
+TODO ADD IMAGE
```
Comment (smbryan) [inline wiki/math/registration-techniques.md:105]: Do you have something to add here? You can add it to the assets folder and link to it from there.

File: wiki/math/registration-techniques.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
+title: Registration Techniques in Robotics
```
Comment (smbryan) [inline wiki/math/registration-techniques.md:1]: Just like in the template, this needs `---` around it for Jekyll to understand. For example:
```
---
Title: My title
---
```

**Pull request merged**